airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/schemas/ad_creative_name.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "com.linkedin.ads.SponsoredUpdateCreativeVariables": {
+      "type": "object",
```

Comment:
The type should be also `nullable` as of `type: ["null", "object"]`, please make sure all the types are declared that way.

Reply:
Yes. I will do it.

Reply:
Done.